src/App.tsx

```diff
@@ -1,14 +1,55 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Filter } from './types/FilterEnum';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [filter, setFilter] = useState<Filter>(Filter.all);
+  const [loader, setLoader] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const handleClose = () => {
+    setSelectedTodo(null);
+  };
+
+  const handleSelectTodo = (todo: Todo) => {
+    setSelectedTodo(todo);
+  };
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+  };
+
+  useEffect(() => {
+    setLoader(true);
+    getTodos()
+      .then(setTodos)
+      .finally(() => setLoader(false));
+  }, []);
+
+  const filteredTodos = todos
+    .filter(todo => {
+      if (filter === Filter.active) {
+        return !todo.completed; // Показувати тільки активні завдання
```

Comment:
It is necessary to remove comments everywhere, or if they are really necessary, then they should be only in English
```suggestion
        return !todo.completed;
```